dwave/cloud/client.py

```diff
@@ -530,119 +530,111 @@ def _fetch_solvers(self, name=None):
-
-        Filtering forms:
-            <inferred_feature> (bool),
-            <inferred_feature>__eq (bool),
-            <inferred_feature>__<operator> (object <value>):
-                Ensures the value of solver's property bound to `inferred_feature`,
-                after applying `operator` equals the `value`. The default
-                operator is `eq`.
-
-            <parameter> (bool),
-            <parameter>__available (bool),
-            <parameter>__<operator> (object <value>):
-                Ensures solver supports `parameter`. General operator form can
-                be used, but that usually doesn't make sense for parameters,
-                since values are human-readable descriptions. The default
-                operator is `available`.
-
-            <property> (bool),
-            <property>__eq (bool),
-            <property>__<operator> (object <value>):
-                Ensures the value of solver's `property`, after applying
-                `operator` equals the righthand side `value`. The default
-                operator is `eq`.
-
-            Note: if a non-existing parameter/property name/key given, the
-            default operator is `eq`.
-
-        Operators:
-            available, eq, lt, lte, gt, gte, regex,
-            covers, within,
-            in, contains,
-            issubset, issuperset
-
-        Inferred features:
-            name (str):
-                Solver name/id.
-            qpu (bool):
-                Is solver QPU based?
-            software (bool):
-                Is solver software based?
-            online (bool, default=True):
-                Is solver online?
-            num_qubits (int):
-                Number of qubits available.
-            num_active_qubits (int):
-                Number of active qubits. Less then or equal to `num_qubits`.
-
-        Common solver parameters:
-            flux_biases:
-                Should solver accept flux biases?
-            anneal_schedule:
-                Should solver accept anneal schedule?
-
-        Common solver properties:
-            vfyc (bool):
-                Should solver work on "virtual full-yield chip"?
-            max_anneal_schedule_points (int):
-                Piecewise linear annealing schedule points.
-            h_range ([int,int]), j_range ([int,int]):
-                Biases/couplings values range.
-            num_reads_range ([int,int]):
-                Range of allowed values for `num_reads` parameter.
+                Force refresh of cached list of solvers/properties.
+            filters:
+                See `Filtering forms` and `Operators` below.
+
+        Solver filters are defined, similarly to Django QuerySet filters, with
+        keyword arguments of form `<name>__<operator>=<value>`. Each `<operator>`
+        is a predicate (boolean) function that acts on two arguments: value of feature
+        `<name>` and the required `<value>`.
+
+        Feature `<name>` can be:
```

Comment:
Using interpreted quote in sphinx will not resolve <name>, since it's not defined anywhere... That's why I like to use double ticks, or alternatively italics for such symbols.

Reply:
We differ there. I find that in RTD the double quotes are too prominent and resemble the clickable links to other methods/classes, and that will be frustrating for users:
![image](https://user-images.githubusercontent.com/34041130/50367165-2f818880-0533-11e9-8514-d0200cc8bf31.png)